src/shipping/handlers.ts

```diff
@@ -45,12 +47,22 @@ export const createChannelsChangeHandler = (
 
 export function getCreateShippingPriceRateVariables(
   data: ShippingZoneRatesPageFormData,
-  id: string
+  id: string,
+  addPostalCodeRules: any,
```

Comment:
Why any?

Reply:
Rush. Changed everywhere to use proper typing.